ceph_deploy/hosts/__init__.py

```diff
@@ -54,7 +54,7 @@ def get(hostname,
         if 'already closed' in getattr(error, 'message', ''):
             raise RuntimeError('remote connection got closed, ensure ``requiretty`` is disabled for %s' % hostname)
     distro_name, release, codename = conn.remote_module.platform_information()
-    if not codename or not _get_distro(distro_name):
```

Comment:
not checking for `codename` is going to cause issues for other non-oracle targets. Maybe a one-off check for oracle might be better?